src/go/rpk/pkg/cli/cmd/acl.go

```diff
@@ -65,3 +80,210 @@ func NewACLCommand(fs afero.Fs, mgr config.Manager) *cobra.Command {
 	command.AddCommand(acl.NewUserCommand(fs))
 	return command
 }
+
+const helpACLs = `Manage ACLs and SASL users.
+
+This command space creates, lists, and deletes ACLs, as well as creates SASL
+users. The help text below is specific to ACLs. To learn about SASL users,
+check the help text under the "user" command.
+
+When using SASL, ACLs allow or deny you access to certain requests. The
+"create", "delete", and "list" commands help you manage your ACLs.
+
+An ACL is made up of five components:
+
+  * a principal (the user)
+  * a host the principal runs on
+  * what resource to access (topic name, group ID, ...)
+  * the operation (read, write, ...)
+  * the permission: whether to allow or deny the above
+
+To specify a single ACL, you need to specify all five. All ACL commands combine
+the permission into the principal and host flags, meaning to specify a single
+ACL, you need four flags.
+
+ACL commands work on a multiplicative basis. If creating, specifying two
+principals and two permissions creates four ACLs: both permissions for the
+first user, as well as both permissions for the second user. Adding two
```

Comment:
I think we shouldn't use the word "user" interchangeably, since a principal can also be an LDAP group (although I'm not sure if it's supported yet in RP): https://docs.confluent.io/platform/current/kafka/authorization.html#principal

```suggestion
first principal, as well as both permissions for the second principal. Adding two
```